Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/util/HelperUtil.java

```diff
@@ -31,12 +31,23 @@
 import java.util.Date;
 import java.util.HashMap;
 import java.util.Map;
+import javax.crypto.Cipher;
+import javax.crypto.spec.SecretKeySpec;
+import javax.faces.application.FacesMessage;
+import javax.faces.context.FacesContext;
+import javax.servlet.http.HttpSession;
+import org.apache.commons.lang3.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 /**
  * @author Tran Tang
  *
  */
 public class HelperUtil {
+    private static final Logger LOG = LoggerFactory.getLogger(HelperUtil.class);
+    private static final String ENCRYPTION_KEY = "MZygpewJsCpRrfOr";
+    private static final String CRYPTO_VALUE = "Blowfish";
```

Comment:
Try to use AES algorithm since it has been adopted by gov and worldwide

Reply:
we discuss this offline and revisit it in a separate ticket